csv2docx/cli.py

```diff
@@ -15,5 +15,13 @@
     '--delimiter', '-d',
     default=";",
     help='delimiter used in your csv. Default is \';\'')
-def main(data, template, delimiter):
-	c2d.convert(data, template, delimiter)
+@click.option(
+    '--name', '-n',
+    required=True,
+    help='naming scheme for output files.')
+@click.option(
+    '--path', '-p',
+    default="output",
+    help='The location to store the files.')
+def main(data, template, delimiter, name, path):
+	csv2docx.convert(data, template, delimiter, name, path)
```

Comment:
The proposed change below requires this to be reordered as:

data, template, path, delimiter, name

Reply:
we have put it in order like:
data, template, custom_name, path, delimiter
Because we made custom_name required that doesn't have any default argument
and all non-default argument comes first.

Reply:
Another good point: `name` should be renamed to `custom_name` to mirror the library.